server/models/groups/createGroup.js

```diff
@@ -0,0 +1,53 @@
+const { pool } = require('../database')
+
+/**
+ *z
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {Object} param0.body
+ * @return {Promise}
+ *
+ */
+
+function createGroup({ username, body }) {
+  return new Promise((resolve, reject) => {
+    const { group_name: groupName, confidential, members } = body
+    pool.query(
+      `INSERT INTO cl_judge.groups (group_name,creator,confidential,member_count)
+        VALUES(?,?,?,?)`,
+      [groupName, username, confidential, members.length + 1],
```

Comment:
`member_count` should be updated only after all members have been added. 
Suppose I gave 10 usernames, out of which only 7 were valid. So, I would add only those 7 usernames which were valid. And remaining would not be added. So, the member_count would be 7 only.